pkg/sdk/go/sdk.go

```diff
@@ -1249,6 +1250,8 @@ func (pm PageMetadata) query() (string, error) {
 	if pm.Permission != "" {
 		q.Add("permission", pm.Permission)
 	}
-
+	if pm.ListPermissions != "" {
+		q.Add("list_permissions", pm.ListPermissions)
```

Comment:
```suggestion
		q.Add("list_perms", pm.ListPermissions)
```

https://github.com/absmach/magistrala/blob/39999ad5ee3acd556083f96f50571744dfd48260/internal/api/common.go#L43